exercises/retree/canonical-data.json

```
@@ -0,0 +1,66 @@
+{
+    "exercise": "retree",
```

```suggestion
  "exercise": "retree",
```
Since GitHub does not yet support multi-line suggestions, perhaps you can view this as a token of @ErikSchierboom's request to indent the file consistently with two spaces. This appears to be the final request for change before we can merge the exercise.